corehq/form_processor/models.py

```diff
@@ -423,7 +423,11 @@ def read_content(self, stream=False):
         try:
             blob = db.get(self.blob_id, self.blobdb_bucket())
         except (KeyError, NotFound, BadName):
-            raise AttachmentNotFound(self.name)
+            try:
+                # http://manage.dimagi.com/default.asp?239638
+                blob = db.get(self.blob_id, self.blobdb_bucket(True))
```

Comment:
might want to check if `self.blob_bucket` is empty and `'-' in str(self.attachment_id)` before hitting riak again